### PR TITLE
Animations part2

### LINKED ISSUE #100: Animations - part 2
Add animations for the following: 

- [x] Moon should subtly rock back and forth 
- [x] Hovering over a button should increase the size a tiny bit
- [x] Clicking on a button should decrease the size a tiny bit 
- [ ] Car exhaust (see [Figma](https://www.figma.com/file/U3WzHiuIg0Mq4eQk2RPb7S/Design-System?node-id=228%3A377&t=CysotpsjxX8DIhBP-0)) should appear one at a time to mimic the movement of exhaust

### PR DESCRIPTION
**Pull request by mike10911**
Added button resizing, moon rocking, and van exhaust animations 

Fixes #100 

Changelist:

- Added `CovidSection.animations.tsx` which contains all covid section animations 
- Using framer, added a moon rocking animation
- Using framer, buttons scale up when hovered and scale down when tapped
- Using framer, animated exhaust coming out of the van

Notes:

@srazh I tried a lot of different looks for the exhaust. Let me know if you want more puffs bc I wasn't sure if you wanted me to only use the 3 puffs that were on Figma. Also let me know if you want me to change the speed, distance, acceleration, size, etc. 

Tested:

localhost on chrome and safari

Screenshots & Screencasts:

### Button resizing on hover and tap

https://user-images.githubusercontent.com/103906123/208589173-8eab5880-a6dc-4b09-82a3-6dcf40834f2c.mov

### Moon rocking

https://user-images.githubusercontent.com/103906123/208589230-247843b7-9983-46c6-a95d-109ff20d49aa.mov

### Van exhaust

https://user-images.githubusercontent.com/103906123/208589275-9b9c52e4-c2d2-49e3-93e8-20a9eff3ccec.mov

### CODE REVIEW

File: main-site/components/covid-section/CovidSection.animations.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+export const smallPuff = {
```
Comment (jzhzhang66) [inline main-site/components/covid-section/CovidSection.animations.tsx]: Is there a reason this file extension is tsx? If not lets keep it .ts like the other animation files

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
OVERALL LOOKS GREAT THANK YOU ANIMATION KINGGGG

A few small comments: 
- At the mid screensizes where the smoke is like literally below the entire van I wonder if we should just not show the animation - what do you and @srazh think?
Screenshot 2022-12-20 at 2 53 21 PM
- On mobile / when theres no van make sure you don't display the exhaust
Screenshot 2022-12-20 at 2 54 14 PM
- For the buttons, can we make them grow + shrink at a quicker pace? Especially for the click animation, since people usually click pretty fast they're not going to see that animation unless they hold down their mouse for like an entire second

**Review by srazh — CHANGES_REQUESTED** [review]
+1 to what Judy said. The exhaust looks great!! It is exactly what I envisioned.

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
For some reason there's an extra exhaust here lOL on mobile
Screenshot 2022-12-21 at 11 26 24 PM

Also could you please rename CovidSection.animations to have the .ts extnesion? Ready to approve after!

**Review by jzhzhang66 — APPROVED** [review]
LGTM

**Review by srazh — APPROVED** [review]
wowww im in awe. lgtm mike!

**Pull request merged**